cmake/utils.cmake

```diff
@@ -263,3 +263,55 @@ function(set_python_module_property target)
         )
     endif()
 endfunction()
+
+# This fuction will populate "out_var" with the default values external
+# projects are expected to need.
+# It can take an optional argument that will replace the list separator
+# in CMake values. For instance, if an external project uses a different
+# list separator, the values in here must be changed to reflect this.
+function(get_external_project_default_values out_var)
+    # Some of these variables might end up not being used by some projects
+    # Therefore avoid useless warnings in the log.
+    list(APPEND setting_list --no-warn-unused-cli)
+
+    # Force the use of ABI version 0 on Linux.
+    # This is what Maya has been using for 2019...2023
+    if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+        list(APPEND setting_list -D_GLIBCXX_USE_CXX11_ABI=0)
```

Comment:
This was supposed to change for 2024, I don't know what is the current status of it.

Reply:
Yes I'm aware of this. Its part of the support for RHEL8. We'll need changes for USD and MayaUsd at some point.